routes/notifications.js

```diff
@@ -0,0 +1,47 @@
+const express = require('express');
+const _ = require('lodash');
+
+const redis = require('../helpers/redis');
+const validTokenMiddleware = require('../middlewares/expoToken');
+
+const router = express.Router();
+
+router.get('/', async (req, res) => {
```

Comment:
I think receiving notifications with http transport is not used anymore, so we may not need this route. Can you confirm @jm90m ?